Resources/Prototypes/Entities/Mobs/Player/dragon.yml

```diff
@@ -0,0 +1,121 @@
+- type: entity
+  name: space dragon
+  id: MobDragon
+  description: A flying leviathan, loosely related to space carps. Go ahead, ask it why it can talk.
+  components:
+  - type: Damageable
+    damageContainer: Biological
+  - type: GhostTakeoverAvailable
+    makeSentient: true
+    name: Space dragon!
+    description: Crash, roast, flood the station with carps!
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Physics
+    bodyType: KinematicController
+  - type: HeatResistance
+  - type: CombatMode
+    disarmAction:
+      name: Disarm
+      enabled: false
+      autoPopulate: false
+  - type: Internals
+  - type: Examiner
+  - type: Speech
+  - type: DoAfter
+  # You can't pull a huge fuckoff dragon right? Butcher him on the spot!
+  #- type: Pullable
+  - type: Actions
+  - type: Input
+  - type: Alerts
+  - type: PlayerMobMover
+  - type: PlayerInputMover
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 5
+    baseSprintSpeed : 5
+  - type: Sprite
+    drawdepth: Mobs
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: alive
+      sprite: Mobs/Aliens/Carps/dragon.rsi
+    noRot: true
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeCircle
+        radius: 0.40
+      mass: 50
+      mask:
+      - Impassable
+      - MobImpassable
+      - SmallImpassable
+      layer:
+      - Opaque
+  - type: MobState
+    thresholds:
+      0: !type:NormalMobState {}
+      450: !type:CriticalMobState {}
+      500: !type:DeadMobState {}
+  - type: SolutionContainerManager
+  - type: Metabolizer
+    solutionOnBody: false
+    updateFrequency: 0.25
+    metabolizerTypes: [Dragon]
+    groups:
+    - id: Medicine
+    - id: Poison
+  - type: HealthExaminable
+    examinableTypes:
+    - Blunt
+    - Slash
+    - Piercing
+    - Heat
+    - Shock
+  - type: Bloodstream
+    bloodlossDamage:
+      types:
+        Bloodloss:
+          1
+    bloodlossHealDamage:
+      types:
+        Bloodloss:
+          -1
+  - type: MovementIgnoreGravity
+  - type: Appearance
+    visuals:
+    - type: DamageStateVisualizer
+      normal: alive
+      crit: crit
+      dead: dead
+  - type: Butcherable
+    spawned:
+    - id: FoodMeatDragon
+      amount: 2
+  - type: UnarmedCombat
+    range: 0.8
+    arcwidth: 2
+    arc: bite
+    damage:
+      groups:
+        Brute: 45
+  - type: Dragon
+    carpProto: MobCarpSentient
+    devourAction:
+      event: !type:DevourActionEvent
+      icon: Interface/Actions/devour.png
+      name: action-name-devour
+      description: action-description-devour
+      devourChemical: Ichor
+      devourHealRate: 15.0
+      whitelist:
+        components:
+        - MobState
+        - Tag
+    carpBirthAction:
+      event: !type:CarpBirthEvent
+      icon: Interface/Actions/carpbirth.png
+      name: action-name-carp-birth
+      description: action-description-carp-birth
+      useDelay: 2
```

Comment:
this should have a higher delay, maybe 30secs

Reply:
It's limited by summon tokens so you won't have an ability to spam it constantly.